linodecli/plugins/obj/buckets.py

```diff
@@ -0,0 +1,78 @@
+# pylint: disable=too-many-lines
+"""
+CLI Plugin for handling OBJ
```

Comment:
nit: should the comment be CLI Plugin for handling buckets, or obj buckets?

Reply:
Good catch! Thank you for the suggestion, and I just updated the doc string accordingly.